app/(golems)/golems/components/Header.tsx

```
@@ -226,14 +231,15 @@ export default function Header() {
         )}
       </header>
 
-      {/* Mobile overlay to close menu */}
+      {/* Mobile overlay to close menu — preventDefault stops phantom taps */}
       {mobileOpen && (
         <div
           className="fixed inset-0 z-40 md:hidden"
           role="button"
           tabIndex={0}
```

Biome (2.3.14)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)